src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
Yeah agreed. actually we should probably rename this component from `LoginHeader` to `ModalHeader` after we start working on the modal component